app/routes/api+/user.prefs.hide-install-pwa-prompt-modal.ts

```diff
@@ -0,0 +1,26 @@
+import { type ActionFunctionArgs, json } from "@remix-run/node";
+import { setCookie, userPrefs } from "~/utils/cookies.server";
+import { makeShelfError } from "~/utils/error";
+import { data, error } from "~/utils/http.server";
+
+export async function action({ context, request }: ActionFunctionArgs) {
+  const authSession = context.getSession();
+  const { userId } = authSession;
+
+  try {
+    const cookieHeader = request.headers.get("Cookie");
+    const cookie = (await userPrefs.parse(cookieHeader)) || {};
```

Comment:
I see you are using the `userPrefs` cookie, but I think we need to create a different one, as we want it to expire after 14 days so we can remind the user again about it.